scripts/syntax.sh

```diff
@@ -3,9 +3,15 @@
 set -ex -o pipefail
 
 DUMMY_GRAMMARS=$(find syntaxes/test -name '*-dummy.json' -exec echo "-g {}" \; | tr '\n' ' ')
+ARGS=$(cat<<ARGS
+  -s template.ng 
+  -g syntaxes/template.ng.json $DUMMY_GRAMMARS 
+  -t syntaxes/test/**/*.ts
+ARGS
+)
 
-# Template syntax tests
-yarn vscode-tmgrammar-test \
-  -s template.ng \
-  -g syntaxes/template.ng.json $DUMMY_GRAMMARS \
-  -t "syntaxes/test/**/*.ts"
+# Unit tests
+yarn vscode-tmgrammar-test $ARGS
+
+# Snapshot tests
+yarn vscode-tmgrammar-snap $ARGS
```

Comment:
Does this generate the snapshot or compares the output with an existing snapshot?
Could you please add a comment somewhere on how to update the snapshot in case a test fails?

Reply:
It compares the output with an existing snapshot. I've updated the script to take a '-u' argument to update the snapshots.